manim/cli/render/commands.py

```diff
@@ -136,4 +137,17 @@ def __repr__(self):
             except Exception:
                 console.print_exception()
 
+    if config.notify_outdated_version or args["notify_outdated_version"]:
```

Comment:
still, `args["notify_outdated_version"]` isn't actually necessary here if the config is doing it's job properly

Reply:
The user may want the default to be `False` and explicitly check using the flag.

Reply:
The config file will handle the explicit check of the flag if it's passed in.
Suppose `manim.cfg` (users shouldn't modify `default.cfg` as it will change on update) has:
```
[CLI]
notify_outdated_version = False
```
Running `manim -pql ...` will render the file and mute the warning from being outdated due to the presence of a manim.cfg file.
If for some reason the user needed to run `manim -pql --notify_outdated_version ... ` (wow that's a long flag name btw), the `config.digest_arg(args)` before any of the rendering will handle the case of explicitly checking the flag. Again, if it doesn't it's a bug.

Reply:
Also, as you might have noticed, some of the other flags in `@global_options, @output_options, @render_options, @ease_of_access_options` are missing the `default = None` that was critical to this option being usable with the config.
This is a relatively easy bug fix that should be done in a separate PR, but really the CLI just needs more exhaustive test cases.